modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -1,16 +1,17 @@
 = Installing and Configuring the External Storage: Windows Network Drives App
-:toc: right
+:toc: macro
```

Comment:
Why change the style from `right` to `macro`? With this change, the table of contents no longer renders in the page.

Reply:
I got it from the occ command, did not rly know why.